README.md

```diff
@@ -39,55 +73,206 @@ tools (such as `ceph-installer` and `gdeploy`):
+    to the values you specify there when you use multiple network interfaces
+    on the machines.
+
+    Note: you can define these variables anywhere else you like (eg. in
+    variable files or from command line directly), but including them into the
+    inventory provides you with a single file with full description of
+    tendrl-ansible setup for future reference (eg. reruning tendrl-ansible
+    later when you need to expand cluster or make sure the configuration still
+    holds).
+
+4)  Add optional ansible variables into the invenory file.
+
+    Based on Tendrl documentation and description in README files of
+    tendrl-ansible roles, specify values for variables you like to tweak.
+
+    This is important because some features tendrl-ansible can help you
+    with are disabled by default as they require additional user input.
+
+    This includes etcd tls client authentication and tendrl notifier
+    configuration for snmp or smtp. Other features such as firewalld setup
+    for Tendrl can be disabled if needed.
+
+5)  If you use tendrl-ansible from rpm package, copy `site.yml` playbook into
+    local directory (where you already store the inventory file):
+
+    ```
+    $ cp /usr/share/doc/tendrl-ansible-VERSION/site.yml site.yml
+    ```
+
+    Do the same for prechecks playbook:
+
+    ```
+    $ cp /usr/share/doc/tendrl-ansible-VERSION/prechecks.yml prechecks.yml
+    ```
+
+6)  Check that ssh can connect to all machines from the inventory file without
+    asking for password or validation of public key by running:
+
+    ```
+    $ ansible -i inventory_file -m ping all
+    ```
+
+    You should see ansible to show `"pong"` message for all machines.
+    In case of any problems, you need to fix it before going on. If you are not
+    sure what's wrong, consult documentation of ansible and/or ssh.
+
+7)  Now you can run prechecks playbook to verify if minimal requirements and
+    setup for Tendrl are satisfied. Any problem with the pre checks will make
+    the playbook run fail immediately, pointing you to a particular
+    requirement or problem with configuration before the installation itself
+    (preventing you to spend time with unnecessary debugging after
+    installation).
+
+    For **production deployment**, run the full check:
+
+    ```
+    $ ansible-playbook -i inventory_file prechecks.yml
+    ```
+
+    While for **proof of concept deployments**, you can avoid checking of
+    stringent production requirements using `production` tag:
+
+    ```
+    $ ansible-playbook -i inventory_file prechecks.yml --skip-tags "production"
+    ```
+
+    If you are not sure why a particular check is there or what is checked
+    exactly, open the playbook file and see comments and/or implementation of
+    the check.
+
+8)  Then we are ready to run ansible to install Tendrl:
+
+    ```
+    $ ansible-playbook -i inventory_file site.yml
+    ```
+
+    Assuming we have deployed ssh keys on the machines and have a cluster
```

Comment:
The note about ssh keys is already mentioned in the step 6, so I'm not sure if it is required here?
And *cluster already installed* means Gluster cluster?

Reply:
Repetition in this context is not a bad thing, but good point about gluster, fix: c48bc68685e2ab7f355c07679f45333de13d6e88